websites/M/myCANAL/presence.ts

```diff
@@ -6,62 +13,110 @@ const presence = new Presence({
 		pause: "general.paused",
 		live: "general.live",
 	}),
-	browsingTimestamp = Math.floor(Date.now() / 1000);
-let title;
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	containsTerm = (term: string) => {
+		if (document.location.pathname.includes(term))
+			return document.location.pathname;
+	};
```

Comment:
Can't this just be

```suggestion
	containsTerm = (term: string) => document.location.pathname.includes(term);
```

Reply:
Hello! This suggestion returns a boolean and you cannot pass a bool type in the case statement

Reply:
If you do `switch (true)` you can